workspace_host/interface.sql

```diff
@@ -29,20 +29,24 @@ WHERE
 
 -- BLOCK insert_workspace_hosts
 INSERT INTO workspace_hosts
-        (instance_id,  hostname)
-VALUES ($instance_id, $hostname)
+        (instance_id,  hostname, state, state_changed_at, ready_at)
+VALUES ($instance_id, $hostname, 'ready', NOW(), NOW())
 ON CONFLICT (instance_id) DO UPDATE
-SET hostname = EXCLUDED.hostname;
+SET hostname = EXCLUDED.hostname,
+    state = EXCLUDED.state,
+    state_changed_at = EXCLUDED.state_changed_at,
+    ready_at = EXCLUDED.ready_at;
```

Comment:
Could be beneficial to leave it at the upsert in case we manually are launching more hosts without having them in the db.  Dunno, don't feel very strongly either way here.

Reply:
Good point, I like the ability to keep manual interventions working.